oak_functions/proto/response.proto

```diff
@@ -0,0 +1,42 @@
+//
+// Copyright 2021 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+syntax = "proto3";
+
+package oak.functions.response;
+
+// Represents the response from an Oak Functions application. This is included in the body of the
+// HTTP response sent to the client.
+message FunctionsResponse {
+  // Status code indicating the success or failure of the operation.
+  FunctionsStatusCode status = 1;
+  // The body of the response generated by the Wasm module, in case of success. Or the reason of the
+  // failure in case of an error.
+  bytes body = 2;
+}
+
+// Status codes for the response.
+enum FunctionsStatusCode {
+  UNSPECIFIED = 0;
+  // Indicates success of the operation. Similar to HTTP 200 status code.
+  SUCCESS = 1;
+  // Indicates a problem with the request. Similar to HTTP 400 status code.
+  BAD_REQUEST = 2;
+  // Indicates violation of the security policy.
+  POLICY_VIOLATION = 3;
```

Comment:
I think we want more precise errors for policy, right? Surely if we are querying a server and it fails because of policy, we would want to know if it's because of time or size.

Reply:
Done.